python/semantic_kernel/memory/memory_record.py

```diff
@@ -53,10 +53,6 @@ def __init__(
         self._additional_metadata = additional_metadata
         self._embedding = embedding
 
-    @property
```

Comment:
why was this property removed?

Reply:
I haven't removed this property, I have added properties for all the variables of the memory_record class in the end of the file.